isis/CMakeLists.txt

```diff
@@ -73,8 +73,9 @@ option(buildCore       "Build the core ISIS modules"                    ON  )
 option(buildMissions   "Build the mission specific modules"             ON  )
 option(buildStaticCore "Build libisis static as well as dynamic"        OFF )
 option(buildTests      "Set up unit, application, and module tests."    ON  )
+option(buildDocs       "Build the docs."                                ON  )
```

Comment:
Building the docs can take a long time and they are not built every time, might be better to leave off by default